src/tests/postgresql/alter/alter-index.test.ts

```diff
@@ -0,0 +1,11 @@
+import {parsePostgreSqlQueryWithCursor} from '../../lib';
+import {KeywordSuggestion} from '../../../types';
+
+test('should suggest properly after ALTER INDEX', () => {
+    const parseResult = parsePostgreSqlQueryWithCursor('ALTER INDEX |');
+
+    const keywordsSuggestion: KeywordSuggestion[] = [{value: 'IF'}, {value: 'ALL'}];
+    expect(parseResult.suggestKeywords).toEqual(keywordsSuggestion);
+
+    expect(parseResult.suggestIndexes).toBeTruthy();
```

Comment:
I wouldn't use toBeTruthy 
It's not specific enough
better to use toEqual(true)

Reply:
I don't like the idea of supporting something we'll not use. We have limited amount of resources and should use them wisely! It's not hard to do if we'll support clickhouse indexes retrieval in websql